internal/cmd/config/config.go

```diff
@@ -332,6 +334,24 @@ func Parse(d string) (*Config, error) {
 			}
 			result.Controller.AuthTokenTimeToStaleDuration = t
 		}
+
+		if result.Controller.Database != nil {
+			if result.Controller.Database.MaxOpenConnectionsRaw != nil {
+				switch t := result.Controller.Database.MaxOpenConnectionsRaw.(type) {
```

Comment:
I think this switch should have a `default` case to error on any other type.

Then, you could have a test case testing what happens when you set `max_open_connections` to, say, ["bad"] - It should fall under that default case